script.module.inputstreamhelper/default.py

```diff
@@ -4,7 +4,16 @@
 from __future__ import absolute_import, division, unicode_literals
 import os
 import sys
-sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), 'lib'))  # noqa: E402
-from inputstreamhelper.api import run
+import xbmc
+import xbmcaddon
+
+
+def to_unicode(text, encoding='utf-8'):
```

Comment:
Because `default.py` is a `xbmc.python.script` extension point and has no access to the `xbmc.python.module` extension point at the `lib` directory
That why I need` sys.path.append` in the first place and to support Windows paths with unicode chars like  `C:\Users\Véronique` I need the ` to_unicode` function.

Reply:
Ok, and what does it do when you run it as script?

Reply:
If it is just showing the settings then that could be done by just `xbmcaddon.Addon().openSettings()`

Reply:
It opens an InputStream Helper add-on settings screen with configurable settings like a custom temp directory and a menu item to remove or reinstall Widevine CDM library.

Reply:
But I wanted an icon in the Kodi GUI, so I needed an `xbmc.python.script` extension point.

Reply:
It's not just for opening settings, I also need the `xbmc.python.script` extension point for api calls used by other add-ons:
For instance for manually (re)installing widevine: `RunScript(script.module.inputstreamhelper, widevine_install)`